src/app.js

```diff
@@ -1,9 +1,14 @@
 const Koa = require('koa');
 const logger = require('koa-logger');
 const router = require('./routes');
+const config = require('./config');
+const { errorsMiddleware } = require('./middleware');
 
 const app = new Koa();
-app.use(logger());
+if (!config.env.isTest) {
```

Comment:
I think that in this types of `if` we have to be explicit with the condition.

```suggestion
if (config.env.isDev || config.env.isProd) {
```